src/complexity.jl

```diff
@@ -0,0 +1,36 @@
+export ComplexityMeasure
+export complexity
+export complexity_normalized
+
+"""
+    ComplexityMeasure
+
+Abstract type for (entropy-like) complexity measures.
+"""
+abstract type ComplexityMeasure end
+
+"""
+    complexity(c::ComplexityMeasure, x)
+
+Estimate the complexity measure `c` for input data `x`, where `c` can be any of the
+following measures:
+
+- [`ReverseDispersion`](@ref).
+
```

Comment:
Shall we add the standard 'see [Input data](2ref) for how Entropies.jl expects input types' here as well? Like in `probabilities`.